cmd/image/import/import.go

```diff
@@ -109,14 +114,13 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --object rhel-83-1003
 			klog.Errorf("Provide valid StorageType.. allowable values are [tier1, tier3]")
 			os.Exit(1)
 		}
-
```

Comment:
Any specific reason removing this space here?

Reply:
No, It got removed while removing my debug logs, Added it again in the latest commit.